tests/test_package/test_query_handlers/query_handler_test.py

```diff
@@ -99,14 +99,14 @@ def handle_query_result(self, query_result: QueryResult) -> Union[Continue, Fini
 
 class QueryHandlerWithOneIterationWithNotReleasedChildQueryHandlerContextFactory(UDFQueryHandlerFactory):
 
-    def create(self, parameter: str, query_handler_context: ScopeQueryHandlerContext) -> UDFQueryHandler:
-        return QueryHandlerWithOneIterationWithNotReleasedChildQueryHandlerContext(parameter, query_handler_context)
+    def create(self, parameters: str, query_handler_context: ScopeQueryHandlerContext) -> UDFQueryHandler:
+        return QueryHandlerWithOneIterationWithNotReleasedChildQueryHandlerContext(parameters, query_handler_context)
```

Comment:
reminds me a Welsh train station name:
Llanfairpwllgwyngyllgogerychwyrndrobwllllantysiliogogogoch

Reply:
This is actually a test case, thats why we need the lengthy name to be precise

Reply:
:rofl: